pkgs/development/tools/doctl/default.nix

```diff
@@ -2,15 +2,27 @@
 
 buildGoPackage rec {
   name = "doctl-${version}";
-  version = "1.3.1";
-  rev = "a57555c195d06bc7aa5037af77fde0665ad1231f";
+  version = "${major}.${minor}.${patch}";
+  major = 1;
+  minor = 5;
+  patch = 0;
+  rev = "66f10023539b5e0e27485a34048366ca1133872f";
   goPackagePath = "github.com/digitalocean/doctl";
 
+  excludedPackages = "cmd/doctl-gen-doc cmd/install-doctl cmd/release-doctl";
```

Comment:
Hum, this might need to be the complete package (i.e. `github.com/digitalocean/doctl/cmd/…`) because it's still generating these binaries that are not needed… I'll look into it.